k8s/resources/common/config/loadflow-server-application.yml

```diff
@@ -1,8 +1,2 @@
 loadflow:
   default-provider: OpenLoadFlow
-
-management:
-  endpoints:
-    web:
-      exposure:
-        include: prometheus, health, info
```

Comment:
Why did you delete it?
It's needed to control which parts we initialize & expose, and the docker part use also this file.